marklogic-data-hub-central/ui/src/config/tooltips.config.tsx

```diff
@@ -182,11 +182,11 @@ const ModelingTooltips = {
   relationshipNameInfo: function (entityName) {
     return (
       <span>
-        The name that identifies the relationship between the source and target entity types. The relationship is saved
-        as a entity property in the <strong>{entityName}</strong> entity type.
+        The name that identifies the relationship between the source and target concept. The relationship is saved as a
+        entity property in the <strong>{entityName}</strong> entity type.
         <br />
-        Names must start with a letter and can contain letters, numbers, hyphens, and underscores. Entity property names
-        cannot use the same name as the associated entity type or its existing Entity properties.
+        Names cannot contain whitespaces. Entity property names cannot use the same name as the associated entity type
```

Comment:
I noticed this tooltip appears for the Entity to Entity relationship as well but it should only apply to the Entity to Concept Relationship. Entity to Entity should keep the same restrictions "Names must start with a letter and can contain letters, numbers, hyphens, and underscores"

Reply:
Thanks for the catch, fixed!